inert.js

```diff
@@ -110,6 +110,24 @@ class InertRoot {
    */
   _makeSubtreeUnfocusable(startNode) {
     composedTreeWalk(startNode, (node) => { this._visitNode(node); });
+
+    let activeElement = document.activeElement;
+    if (!document.contains(startNode)) {
```

Comment:
do you know how expensive it is to call this? I haven't worked with `contains` before